websites/S/Snipes/dist/metadata.json

```
@@ -7,17 +7,11 @@
   "service": "Snipes",
   "description": {
     "en": "SNIPES is a store for popular brand clothes.",
-    "de": "SNIPES ist ein Handel für beliebte marken Klamotten."
+    "de": "SNIPES ist ein Handel für beliebte Markenklamotten."
   },
-  "url": [
-    "www.snipes.com",
-    "www.snipes.at",
-    "www.snipes.fr",
-    "www.snipes.nl",
-    "www.snipes.it",
-    "www.snipes.es"
-  ],
-  "version": "1.0.1",
+  "url": "www.snipes.com",
+  "regExp": "REGEXP",
```

Make it yourself if you don't want to use this

```suggestion
  "regExp": "www[.]snipes[.]([a-z]+)",
```